### PR TITLE
feat: Gamify applications admin dashboard

### PR DESCRIPTION
**Pull request by sanderelderman**
Features:
- General UI added to web-app containing a toolbar and sidenav when navigating to an admin route.
- Applications overview and form to create and edit an application.
- Application badges overview with a dialog to create and edit badges.
- Application users overview to manage the badges of an user.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,10 +12,12 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "13.3.4",
+    "@angular/cdk": "^13.2.0",
```
Comment (mschilling) [inline package.json:15]: If this version works: fine. 

Curious to know if there isn't a version `13.3.4` available

**Review by mschilling — APPROVED** [review]
PR approved and focussed in particular on the NestJS changes. 

Tip: try to keep your PR's as small as possible. In this case: it would be helpful to split the frontend and backend work over two PR's (if possible). This way you can assign reviewers for a specific stack.

File: package.json (hunk 1 above)
Comment (cassshh) [reply package.json:15]: There is 😉 (13.3.4 available)

Comment (sanderelderman) [reply package.json]: Updated this package and angular material to 13.3.4

**Pull request merged**